src/main/java/org/cbioportal/service/impl/StudyViewColumnarServiceImpl.java

```diff
@@ -110,5 +112,10 @@ public List<ClinicalData> getSampleClinicalData(StudyViewFilter studyViewFilter,
         return studyViewRepository.getSampleClinicalData(studyViewFilter, attributeIds, categorizedClinicalDataCountFilter);
     }
 
+    @Override
+    public List<GenomicDataCountItem> getMutationTypeCountsByGeneSpecific(StudyViewFilter studyViewFilter, List<GenomicDataFilter> gdFilters) {
+        CategorizedClinicalDataCountFilter categorizedClinicalDataCountFilter = extractClinicalDataCountFilters(studyViewFilter);
+        return studyViewRepository.getMutationCountsByType(studyViewFilter, categorizedClinicalDataCountFilter, gdFilters);
```

Comment:
would be good to put a comment comment here why we need the clinical data count filter